src/main/java/reposense/RepoSense.java

```diff
@@ -34,12 +34,15 @@
     private static final int SERVER_PORT_NUMBER = 9000;
     private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("E MMM d HH:mm:ss yyyy z");
     private static final String VERSION_UNSPECIFIED = "unspecified";
+    private static final String MESSAGE_ELAPSED_TIME = "Elapsed processing time: %f seconds";
 
     /**
      * The entry point of the program.
      */
     public static void main(String[] args) {
         try {
+            long startTime = System.nanoTime();
```

Comment:
Not sure if it is a good place to leave it here.

Reply:
Maybe before the `generateReposReport` method? Since cloning and analysing occurs in that method.
https://github.com/reposense/RepoSense/blob/0848879f9e89f1fac3dea51285a9c503d2554f33/src/main/java/reposense/RepoSense.java#L63-L66

Reply:
What I mean is to have a seperate class for the responsibility of time recording.And call it in main method. But not sure if it going to be too complicated.

Reply:
@fzdy1914 
Interesting suggestion.

If we make it an `interface` then we can track time elapsed for each step in the analysis. 🤔

Reply:
That's true, we can even know the time taken for every repo. However, the priority of that is low. This is more for our developer to know more info about the analysis.